.ado/templates/fluentui-apple-publish-nuget-job.yml

```diff
@@ -28,6 +28,11 @@ jobs:
       packagesToPack: 'FluentUI.nuspec'
       buildProperties: buildNumber=$(sanitizedBuildNumber);commitId=$(Build.SourceVersion);repoUri=$(Build.Repository.Uri)
 
+ - task: AzureArtifacts.manifest-generator-task.manifest-generator-task.ManifestGeneratorTask@0
+   displayName: 'Generation Task'
+   inputs:
+       BuildDropPath: '$(System.ArtifactsDirectory)/buildOutput'
```

Comment:
@dannyvv do you know if this should be ArtifactStagingDirectory ? or left as system.artifactsdirectory?

Reply:
I would use the higher directory, so it will include the nupkg file as well.
Also the manifest will need ot be uploaded somewhere, so I'd recommend adding a special SBom manifest for it:  https://github.com/microsoft/flamegrill/blob/06dc9a74d87858e310ebbda230876f3bf18a5357/azure-pipelines.release.yml#L47

Reply:
hmmm still unsure what is the "higher" directly. applied the manifest.